### ISSUE TITLE
Support complex matrices

### ISSUE THREAD

**Issue by kburns** [yig/PySPQR#8]
It looks like this package may currently just wrap the factorization and solve routines for real-valued matrices (is this right?).  It would be great if it could be extended to also support factorizing and solving complex-valued matrices, as well.

**Comment by yig** [comment]
You are correct. I don't really use complex matrices, so I didn't implement support. Feel free to submit a pull request.

**Comment by donsano33** [comment]
I opened https://github.com/yig/PySPQR/pull/30 which enables the usage of complex matrices (at least for `sparseqr.qr`). I didn't check if this also works for `sparseqr.solve`, but it might.

**Comment by yig** [comment]
I adapted @donsano33 changes to dense matrices, so now solve works.
